tests/test_context_managers.py

```diff
@@ -196,7 +196,7 @@ def test_query_counter(self):
             self.assertEqual(0, q)
 
             for i in range(1, 51):
-                db.test.find({}).count()
+                db.test.estimated_document_count({})
```

Comment:
What does the `{}` argument represent?

Reply:
I can't recall if I originally thought this was passed on as a query argument, but it actually becomes a `comment` parameter, and isn't necessary to retain the original functionality. I've removed it, thanks.